plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/client/PinotGrpcDataFetcher.java

```diff
@@ -219,8 +220,6 @@ public GrpcQueryClient create(HostAndPort hostAndPort)
 
     public static class PinotGrpcServerQueryClient
```

Comment:
Is this object shared across queries? Asking because now they'll all share the same `CalciteSqlCompiler`.

Reply:
Nevermind, `CalciteSqlCompiler` and `CalciteSqlParser` both seem thread-safe - no state at all there.